specification/sql/resource-manager/readme.md

```diff
@@ -121,6 +121,7 @@ input-file:
 - Microsoft.Sql/preview/2020-02-02-preview/managedInstances.json
 - Microsoft.Sql/preview/2020-02-02-preview/ManagedInstanceAzureADOnlyAuthentications.json
 - Microsoft.Sql/preview/2020-02-02-preview/ServerTrustGroups.json
+- Microsoft.Sql/preview/2020-08-01-preview/ServerDevOpsAudit.json
```

Comment:
can you follow the alphabetical order?

Reply:
and please don't merge this PR until your DSMainDev PR deployment.

Reply:
This API already deployed in Pord.
I made some changes following comments on this PR, mostly swagger/doc related.
i don't think any of these changes would be a blocker of merging this.
Also, i am updating current version 2020-08-01-preview, which already merged.